src/plugins/template/src/template_config.cpp

```diff
@@ -22,19 +22,19 @@ Configuration::Configuration(const ConfigMap& config, const Configuration& defau
         const auto& value = c.second;
 
         if (TEMPLATE_CONFIG_KEY(THROUGHPUT_STREAMS) == key) {
-            _streamsExecutorConfig.SetConfig(CONFIG_KEY(CPU_THROUGHPUT_STREAMS), value);
+            _streamsExecutorConfig.SetConfig(CONFIG_KEY(CPU_THROUGHPUT_STREAMS), value.as<std::string>());
         } else if (streamExecutorConfigKeys.end() !=
                    std::find(std::begin(streamExecutorConfigKeys), std::end(streamExecutorConfigKeys), key)) {
-            _streamsExecutorConfig.SetConfig(key, value);
+            _streamsExecutorConfig.SetConfig(key, value.as<std::string>());
         } else if (CONFIG_KEY(DEVICE_ID) == key) {
-            deviceId = std::stoi(value);
+            deviceId = std::stoi(value.as<std::string>());
```

Comment:
looks strange that we convert to string, then parse it.. maybe we need to extract int directly from ov::any?